scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/ActivityType.java

```diff
@@ -37,7 +37,7 @@ public ActivityType(String name) {
    *
    * @param name IN the identifier of the activity type
    */
-  public ActivityType(String name, TaskSpecType<?,?> specType) {
```

Comment:
:small_red_triangle_down: If this change (and it's associated changes to the TaskSpecType type signature) is relocated to an earlier commit f917e5f then it will compile.

Reply:
done